### ISSUE TITLE
video classification from tensors

### ISSUE THREAD

**Issue by fcakyon** [Lightning-AI/lightning-flash#1356]
Hello all!

As far as I understood, it is not possible to perform real-time video classification using flash since it only supports prediction from folders/video files.

If you guide me on how to implement it, I would like to open a PR for performing video classification prediction from tensors.

**Comment by krshrimali** [comment]
Hi, @fcakyon - Thank you for your interest, are you looking for something like:

```python
data_module = VideoClassificationData.from_files(train_files=<list_of_tensors>, ...)
```

Just want to make sure that I got your suggestion right. But in any case, I would love to see a draft PR for the use-case you mentioned, and once we have a green signal on the design, we can take it forward ⚡ 

Let me know if you need any help with the PR. Apologies for the delay in the response. :)

**Comment by ethanwharris** [comment]
Fixed by #1389
